editor/editor_node.cpp

```diff
@@ -197,6 +197,9 @@ void EditorNode::_version_control_menu_option(int p_idx) {
 
 			VersionControlEditorPlugin::get_singleton()->shut_down();
 		} break;
+		default: {
+			show_warning(TTR("No such VCS menu handler. This is a bug, please report"));
```

Comment:
Small nitpick, there's a missing period at the end :slightly_smiling_face:

```suggestion
			show_warning(TTR("No such VCS menu handler. This is a bug, please report."));
```